File: Excercises/Javascript/euler.js
let eNumber = 0;

let counter = 0;
while (counter < 100) {
    // first calculate the factorial
    let factorial = 1;
    let anotherCounter = counter;
    while (anotherCounter > 0) {
        factorial *= anotherCounter;
        anotherCounter--;
    }
    eNumber += 1.0 / (factorial);
    counter++;
}

console.log(eNumber);

File: Excercises/C/euler.c
#include <stdio.h>

int main() {

    int number;
    long double factorial, euler = 1.0;

    printf("Input a number: ");
    scanf("%d", &number);

    int counter = 1;
    while (counter <= number) {
        // factorial
        factorial = 1;
        int anotherCounter = counter;
        while (anotherCounter > 0) {
            factorial *= anotherCounter;
            anotherCounter--;
        }
        // calc of the e number
        euler += (1.0 / factorial);
        counter++;
    }

    printf("The approximation the e number is: %Lf\n", euler);

    return 0;
}
